### PR TITLE
Update Mendix Cloud deployment docs to align with exisiting UI

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/studio-deployment-settings.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+tags: ["node","developer portal","deploy", "Studio", "settings", "target", "Mendix Studio Pro Target"]
 #To update these screenshots, you can log in with credentials detailed in How to Update Screenshots Using Team Apps.
 ---
 
 ## 1 Introduction
 
-If you have a licensed app in the Mendix Cloud which you want to deploy directly to an environment, you need to select a target environment where Mendix Studio and Mendix Studio Pro will deploy your app when you click **Publish** (for Studio) or **Deploy to Licensed Cloud Node** (for Studio Pro). For more information on licensed apps and environments, see [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) and [Environments](/developerportal/deploy/environments/).
+If you have a licensed app in the Mendix Cloud which you want to deploy directly to an environment, you need to select a target environment where Mendix Studio Pro will deploy your app when you **Deploy to Licensed Cloud Node**. For more information on licensed apps and environments, see [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) and [Environments](/developerportal/deploy/environments/).
 
 {{% alert color="info" %}}
```
Comment (Luyao-Zhang-1) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/studio-deployment-settings.md:14]: Do you know if this note is still true? @MarkvanMents

**Comment by Luyao-Zhang-1** [comment]
@MarkvanMents This PR is meant to cover all the Mendix Cloud changes which are already live. I brought the changes for Mendix Cloud deployment docs that you and @MariaShaposhnikova have been discussed in the private repo here. Could you please help me review this PR? If it looks good, I will go ahead and merge it.

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_index.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ A Free App has a number of limitations compared to a licensed app. The main limi
 | App vCPUs | 0.5 | Depends on your [pricing plan](#plans). |
 | Scheduled events | Are not run. | Are run and can be configured from the Developer Portal. |
 | Environments | Single environment in the Mendix Cloud. | A node in the cloud which has one or more environments, for example, production, acceptance, and test. |
-| Deployment | Can only be deployed to the cloud from Mendix Studio or Studio Pro. | Can be deployed from the Studios, or from the Developer Portal. |
+| Deployment | Can only be deployed to the cloud from Mendix Studio Pro. | Can be deployed from the Studio Pro, or from the Developer Portal. |
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/_index.md]:
```suggestion
| Deployment | Can only be deployed to the cloud from Mendix Studio Pro. | Can be deployed from Studio Pro, or from the Developer Portal. |
```

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/studio-deployment-settings.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/studio-deployment-settings.md:14]: I think it is. I think we should leave it in for now, but perhaps ask in Slack and update later if it turns out to be wrong.

**Review by MarkvanMents — APPROVED** [review]
Looks good @Luyao-Zhang-1 
Picked up one typo which I'll commit separately.
See my comments, but I will approve this and it can be merged - we can pick up any further questions later.
Thanks
Mark

**Pull request merged**